Modules/fcntlmodule.c

```diff
@@ -66,6 +66,11 @@ fcntl_fcntl_impl(PyObject *module, int fd, int code, PyObject *arg)
     char buf[1024];
     int async_err = 0;
 
+    if (PySys_Audit("fcntl.fcntl", "iiO", fd, code,
+                    arg ? arg : PyLong_FromLong(0)) < 0) {
```

Comment:
```suggestion
                    arg ? arg : Py_None) < 0) {
```

Otherwise we will cause a reference leak - the `PySys_Audit` call will do its own incref/decref on the arguments, which will leave our PyLong_FromLong reference behind on return.